.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Release CI
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Create Build
+        run: npm run build
+
+      - name: Release Package
+        uses: cycjimmy/semantic-release-action@v2
+        with:
+          semantic_version: 16
+        env:
+          NPM_TOKEN: ${{ secrets.SEMANTIC_RELEASE_NPM_TOKEN }}
+
+      - name: Send failure notification
+        if: ${{ failure() }}
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: email-smtp.us-east-1.amazonaws.com
+          server_port: 465
+          username: ${{ secrets.EDX_SMTP_USERNAME }}
+          password: ${{ secrets.EDX_SMTP_PASSWORD }}
+          subject: NPM Release Failure
+          to: oscm@edx.org
+          from: github-actions <github-actions@edx.org>
+          body: NPM release workflow in ${{github.repository}} failed! For details see "github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Comment:
I would probably leave this failure notification off for this repo. This repository isn't really sensitive enough for open-source support effort to warrant a notification if this failed to publish to npm. Hopefully the person who merged it would notice, however.

Reply:
Updated the CI to remove the notification step as per the suggestion.